Brauser.Common/Recipe/Hop.cs

```diff
@@ -5,6 +5,6 @@ public record Hop
     public int? Order { get; set; }
     public string? Variety { get; set; }
     public double? Quantity { get; set; }
-    public int? Alpha { get; set; }
-    public DateTimeOffset? CookingTime { get; set; }
+    public double? Alpha { get; set; }
+    public int? CookingTime { get; set; }
```

Comment:
Do we really want an `int` here? It's our business model, so we can decide if we want to convert it to something useful like `DateTimeOffset` after importing the MaischeMalzUndMehr recipe.

Reply:
Fair point. Maybe we could rename it in CookingTimeInMinutes.
DateTimeOffset is nice, but it seems we exceed an hour in very few occasions and seems a little bit overkill in that case.

Reply:
`DateTimeOffset` would not be a good candidate, as it is just a date and says nothing about the duration. My proposal would be `TimeSpan` here. But `CookingTimeInMinutes` would be fine as well. 👍

Reply:
Didn't have TimeSpan on the screen. Nice!